students/Roy_Tate/lesson03/slice_lab.py

```diff
@@ -0,0 +1,155 @@
+# Author/Student: Roy Tate (githubtater)
+
+
+str_sequence = "Humpty Dumpty sat and watched a film"
+num_sequence = 1, 2, 3, 4, 5, 6, 7
+
+a_string = "this is a string"
+a_tuple = (2, 54, 13, 12, 5, 32)
+
+
+def exchange_first_last(sequence):
+    first = sequence[0]
+    print('first: ', first)
+    last = sequence[-1]
+    print('last: ', last)
+    middle = sequence[1:len(sequence) -1]
+    print(middle)
+    new_sequence = last + middle + first
+    return new_sequence
```

Comment:
try using _only_ slicing here (`[-1]` <- is not slicing)

hint: this can be a simple one liner